docs/getting-started.md

```diff
@@ -99,58 +127,99 @@ This will generate `build/classes/java/main/com/org1/contract/StateUpdater.class
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------ | ----------------------------------------------- |
+| [JacksonBasedContract](https://scalar-labs.github.io/scalardl/javadoc/latest/ledger/com/scalar/dl/ledger/contract/JacksonBasedContract.html) (recommended) | [JsonNode](https://fasterxml.github.io/jackson-databind/javadoc/2.13/com/fasterxml/jackson/databind/JsonNode.html) | [Jackson](https://github.com/FasterXML/jackson) |
+| [JsonpBasedContract](https://scalar-labs.github.io/scalardl/javadoc/latest/ledger/com/scalar/dl/ledger/contract/JsonpBasedContract.html)                   | [JsonObject](https://javadoc.io/static/javax.json/javax.json-api/1.1.4/javax/json/JsonObject.html)                 | [JSONP](https://javaee.github.io/jsonp/)        |
+| [StringBasedContract](https://scalar-labs.github.io/scalardl/javadoc/latest/ledger/com/scalar/dl/ledger/contract/StringBasedContract.html)                 | [String](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html)                                          | Java Standard Libraries                         |
+| [Contract](https://scalar-labs.github.io/scalardl/javadoc/latest/ledger/com/scalar/dl/ledger/contract/Contract.html) (deprecated)                          | [JsonObject](https://javadoc.io/static/javax.json/javax.json-api/1.1.4/javax/json/JsonObject.html)                 | [JSONP](https://javaee.github.io/jsonp/)        |
 
-To quickly run and test your contrats in your local environment, [Scalar DL Emulator](https://github.com/scalar-labs/scalardl-tools/tree/master/emulator) might be useful. It uses a mutable in-memory ledger instead of an immutable ledger database and provides an interactive interface, making it easy to do trial and error testing.
+
+They differ in the way to serialize and deserialize arguments and data that you read and write via Contract and Ledger interfaces. JacksonBasedContract uses [Jackson](https://github.com/FasterXML/jackson) to serialize a JSON object (i.e., [JsonNode](https://fasterxml.github.io/jackson-databind/javadoc/2.13/com/fasterxml/jackson/databind/JsonNode.html)) to internal String representation and deserialize a String object to a [JsonNode](https://fasterxml.github.io/jackson-databind/javadoc/2.13/com/fasterxml/jackson/databind/JsonNode.html) object. In the same way, JsonpBasedContract uses [JSONP](https://javaee.github.io/jsonp/) for JSON serialization and deserialization. StringBasedContract does not do JSON serialization and deserialization and uses internal String representation as it is. Using `JacksonBasedContract` is recommended to make a good balance between development productivity and performance, 
```

Comment:
```suggestion
They differ in the way to serialize and deserialize arguments and data that you read and write via Contract and Ledger interfaces. JacksonBasedContract uses [Jackson](https://github.com/FasterXML/jackson) to serialize a JSON object (i.e., [JsonNode](https://fasterxml.github.io/jackson-databind/javadoc/2.13/com/fasterxml/jackson/databind/JsonNode.html)) to internal String representation and deserialize a String object to a [JsonNode](https://fasterxml.github.io/jackson-databind/javadoc/2.13/com/fasterxml/jackson/databind/JsonNode.html) object. In the same way, JsonpBasedContract uses [JSONP](https://javaee.github.io/jsonp/) for JSON serialization and deserialization. StringBasedContract does not do JSON serialization and deserialization and uses internal String representation as it is. Using `JacksonBasedContract` is recommended to make a good balance between development productivity and performance. 
```
This is a very minor point, but it seems that the once fixed statement was overridden by another commit.
(Just changing from , to . in the end of sentence.)

Reply:
Thank you! 🙇